dolphinscheduler-ui-next/src/locales/modules/zh_CN.ts

```diff
@@ -26,7 +26,9 @@ const login = {
 
 const modal = {
   cancel: '取消',
-  confirm: '确定'
+  confirm: '确定',
+  save_error_msg: '保存失败，请重试',
+  delete_error_msg: '删除失败，请重试',
```

Comment:
This message information should not be presented in the modal. This is a business function, and the meaning of the button in the place where the pop-up window is used will be different.